spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -2165,26 +2126,67 @@ public SchemaRegistryClient schemaRegistryClient(@Value("${spring.cloud.stream.s
   return client;
 }
 ----
-NOTE: The ConfluentSchemaRegistryClient is tested against Confluent platform version 3.2.2.
+NOTE: The ConfluentSchemaRegistryClient is tested against Confluent platform version 4.0.0.
 
-==== Schema Registry Client Properties
+=== Apache Avro Message Converters
 
-The Schema Registry Client supports the following properties:
+Spring Cloud Stream provides support for schema-based message converters through its `spring-cloud-stream-schema` module.
+Currently, the only serialization format supported out of the box for schema-based message converters is Apache Avro, with more formats to be added in future versions.
 
-`spring.cloud.stream.schemaRegistryClient.endpoint`:: The location of the schema-server.
-When setting this, use a full URL, including protocol (`http` or `https`) , port, and context path.
-+
-Default:: `http://localhost:8990/`
-`spring.cloud.stream.schemaRegistryClient.cached`:: Whether the client should cache schema server responses.
-Normally set to `false`, as the caching happens in the message converter.
-Clients using the schema registry client should set this to `true`.
-+
-Default:: `true`
+The `spring-cloud-stream-schema` module contains two types of message converters that can be used for Apache Avro serialization:
+
+* Converters that use the class information of the serialized or deserialized objects or a schema with a location known at startup.
+* Converters that use a schema registry. They locate the schemas at runtime and dynamically register new schemas as domain objects evolve.
+
+=== Converters with Schema Support
+
+To use it, you can simply add it to the application context, optionally specifying one or more `MimeTypes` with which to associate it.
```

Comment:
I think it might be useful to define `AvroSchemaMessageConverter` before the usage description.  We can revert to the original order; perhaps we could merge them both into a single statement.